### ISSUE TITLE
Split off Rabbit in a Hat into a new repo

### ISSUE THREAD

**Issue by aguynamedryan** [OHDSI/WhiteRabbit#50]
Though they are closely related, WhiteRabbit and Rabbit in a Hat are two separate programs and it makes better sense in my head to have them in separate repositories.

**Comment by aguynamedryan** [comment]
In a complete 180 of this suggestion, today several of us here at @OutcomesInsights discussed perhaps merging most of WhiteRabbit into Rabbit in a Hat.  As we've discussed before, we're hoping to characterize the data in each column and we're not certain if that's a task best left to WhiteRabbit or RiaH and then we started wondering why the two are separate at all.

Thoughts?

**Comment by schuemie** [comment]
WhiteRabbit and Rabbit in a Hat already share a lot of code. The reason why I created two front ends was for simplicity for the user. I figured that WhiteRabbit needs to be run by the 'tech guy' who mostly cares about getting the database connectivity right. Rabbit in a Hat is intended to be used by the team designing the ETL, who really do not care about database connectivity stuff but need a fancy GUI to make the process a fluid one.
